### DISCUSSION TITLE
blog/2023-06-25-causal-inference-with-beta-regression/

### DISCUSSION THREAD

**Discussion by giscus** [ASKurz/blogdown5#21] [Comments]
# blog/2023-06-25-causal-inference-with-beta-regression/

Part 9 of the GLM and causal inference series.

https://solomonkurz.netlify.app/blog/2023-06-25-causal-inference-with-beta-regression/

**Comment by DrJerryTAO** [comment]
In line with the logit link function used in the beta regression, can we also use a logit transformation of the aPASAT scores that are bounded between zero and one as the dependent variable in an OLS, such as `qlogis(post) ~ 1 + txf + qlogis(pre)` ? This way, we can guarantee that predicted aPASAT scores are always bounded between zero and one after transforming the dependent variable using `plogis()`.

**Reply by ASKurz** [reply]
Maybe. I haven't looked to see whether that approach has been explored in the methods literature before. Speaking for myself, I prefer customizing the likelihood and link function to the data, rather than transforming the data to fit the likelihood.

**Comment by MatthewBJane** [comment]
You are the man!
